modules/roxctl-generate-init-bundle.adoc

```diff
@@ -19,7 +19,7 @@ You can create an init bundle by using the the `roxctl` CLI.
 ----
 $ roxctl -e "$ROX_CENTRAL_ADDRESS" \
   central init-bundles generate <cluster_init_bundle_name> \
-  --output cluster_init_bundle.yaml
```

Comment:
Hm. This file `modules/roxctl-generate-init-bundle.adoc` above says it is referenced in two places:

```
// Module included in the following assemblies:
//
// * installing/installing_helm/install-helm-customization.adoc
// * installing/install-ocp-operator.adoc
```

I suppose `install-helm-customization.adoc` should remain with `--output` but `install-ocp-operator.adoc` should go with `--output-secrets`. I.e. changing it here will break helm doc, keeping it as-is breaks operator doc.
I guess, the way to deal with it is by creating two versions: one with `--output`, another with `--output-secrets`.

Reply:
Good callout. I'll close and modify.